chain/db/pubsub.go

```diff
@@ -99,27 +224,35 @@ func (cs *ChainStore) GetSubscribers(topic string, bucket uint32) map[string]str
 		identifier, _ := serialization.ReadVarString(rk)
 		subscriberString := address.MakeAddressString(subscriber, identifier)
 
-		subscribers[subscriberString] = string(iter.Value())
+		subscribers[subscriberString] = string(iter.Value)
```

Comment:
subscriberString should be stored in `iter.Value` right? The `iter.Key` now contains only the hash

Reply:
yeah, noticed it too and fixed already :)